rifle.py

```diff
@@ -92,6 +92,7 @@ def update(self, events):
 
     def draw(self, screen):
         self.draw_background(screen)
+        #self.draw_friendly_projectiles(screen)
```

Comment:
Yes. This was probably the most confusing merge conflict I ever dealt with and I was very confused whether or not that was used. I guess not xD?